### PR TITLE
Readme reorganized

### PR DESCRIPTION
**Pull request by MrStahlfelge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,22 @@
-# Appkit: A Library for Polyglot Development of Ergo Applications 
+## Appkit: SDK for Ergo Platform dApps, wallet applications, API clients
```
Comment (aslesarenko) [inline README.md:1]:
```suggestion
## Appkit: SDK for writing applications interacting with Ergo blockchain
```

File: README.md [hunk 2]
```diff
@@ -1,38 +1,22 @@
-# Appkit: A Library for Polyglot Development of Ergo Applications 
+## Appkit: SDK for Ergo Platform dApps, wallet applications, API clients 
 
-## Contents
+![Maven Central](http://maven-badges.herokuapp.com/maven-central/org.ergoplatform%20/ergo-appkit_2.12/badge.svg)
```
Comment (aslesarenko) [inline README.md:3]:
```suggestion
![Maven Central](http://maven-badges.herokuapp.com/maven-central/org.ergoplatform%20/ergo-appkit_2.12/badge.svg)
[![CI](https://github.com/ergoplatform/ergo-appkit/actions/workflows/ci.yml/badge.svg)](https://github.com/ergoplatform/ergo-appkit/actions/workflows/ci.yml)
[![Publish a release](https://github.com/ergoplatform/ergo-appkit/actions/workflows/release.yml/badge.svg)](https://github.com/ergoplatform/ergo-appkit/actions/workflows/release.yml)
```

File: README.md [hunk 3]
```diff
@@ -71,300 +55,84 @@ overhead compared to a Java VM. For example this allows to create very
...
+(other examples on the Maven Central page).
 
-```shell
-$ cd <your/directory/with/downloaded/graal>
-$ tar -zxf graalvm-ce-darwin-amd64-19.2.1.tar.gz
-$ export GRAAL_HOME=<your/directory/with/downloaded/graal>/graalvm-ce-19.2.1/Contents/Home
-$ export PATH=$PATH:${GRAAL_HOME}/bin
-```
+As an alternative, you can download the jar file from the 
+[releases page](https://github.com/ergoplatform/ergo-appkit/releases) and add it to your project manually.
```
Comment (aslesarenko) [inline README.md:84]: Jar files are not created for releases.

**Review by aslesarenko — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (MrStahlfelge) [reply README.md:86]: Oh yes they are! https://github.com/ergoplatform/ergo-appkit/blob/315c6d36955756e0f78da63e214a12d4221568b2/.github/workflows/release.yml#L46 - stored on the action, I copy them manually to the releases page each time. Could be done from the action as well.

Comment (aslesarenko) [reply README.md:86]: I looked at the releases, no jars, so maybe first setup auto-attachment of jars?

**Review by aslesarenko — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (MrStahlfelge) [reply README.md:86]: [This release](https://github.com/ergoplatform/ergo-appkit/releases/tag/v4.0.9) has the jar attached, but I can add a change to automatically add it from the action here as well.

**Pull request merged**